apps/docs/src/guide/types/std-string.md

```diff
@@ -2,14 +2,12 @@
 
 A dynamic string of variable length can be represented using the `StdString` type, also known as a Standard Lib String or `std-lib-string`. It behaves much like a dynamic string in most languages, and is essentially an array of characters.
 
-<<< @/../../docs-snippets/src/guide/types/std-string.test.ts#std-string-1{ts:line-numbers}
-
```

Comment:
Should this have been removed?

(Please resolve as confirmation)

Reply:
Same as https://github.com/FuelLabs/fuels-ts/pull/3340#discussion_r1808780178